manifests/package.pp

```diff
@@ -1,16 +1,8 @@
-# Class strongswan::package
+# @summary This class manages Strongswan package installation
 #
-# This module manages Strongswan package installation
-#
-# Parameters:
-#
-# There are no default parameters for this class.
-#
-# Sample Usage:
-#
-# This class file is not called directly
-
+# @api private
 class strongswan::package {
+  include strongswan
```

Comment:
`strongswan::package` is `require`d from some of the other public classes and defined types.  This `include` makes sure that the base classes variables have been resolved.

Reply:
@alexjfisher , since this class is private it must not be called directly, means it is called only from within the strongswan class. That's why on the line 8  $strongswan::package_ensure variable is used. Is then line 5 redundant?

Reply:
It’s private to the module, that doesn’t necessarily mean only the main class can `include` it. The types can include or more likely `require` it if that makes sense. We don’t expect *users* to declare it in their own profiles though.

Reply:
I mean *most* people will probably `include strongswan` themselves before using the defined types, but if they don't they'd get an error they might not understand.  This prevents that. Notice how I've been able to remove all of the `pre_conditions` from the spec tests.